x-pack/plugin/core/template-resources/src/main/resources/traces@settings.json

```diff
@@ -2,6 +2,9 @@
   "template": {
     "settings": {
       "index": {
+        "lifecycle": {
```

Comment:
Should we use DSL instead of ILM given that OTel is new and we don't need tiered lifecycle management?

Reply:
Hmm, but looking at the docs and the API I am not sure if DSL can be configured without retention...

Reply:
> Should we use DSL instead of ILM given that OTel is new and we don't need tiered lifecycle management?

I think we should use one form of lifecycle management consistently for all OTel signals. As we're using ILM by default for metrics and logs, I think we should do the same for traces. For stateful, I think the predominant model will still be ILM, as a lot of self-managed clusters will use a hot/warm/cold architecture. For serverless, we'll override the settings to use DSL by default.

> Hmm, but looking at the docs and the API I am not sure if DSL can be configured without retention...

You can just do something like this:
```json
"lifecycle": {}
```